.github/workflows/test.yml

```diff
@@ -0,0 +1,60 @@
+name: Test
+
+on: [push, pull_request]
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["2.7", "3.5", "3.6", "3.7", "3.8", "3.9", "pypy3"]
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
I don't think macos grants us any additional coverage and the macos builders are pretty slow and in high demand -- after adding pypy we're looking at 24 builders which seems excessive to me

we can probably get by with:
- test all of them on linux
- test py2 + minimum py3 + maximum py3 + pypy + pypy3 on windows

Reply:
Thanks for the review!

Note that pypy is broken on GHA until they include the latest pypy release (see https://github.com/actions/setup-python/issues/163), so pypy2/3 are excluded for now. I've included a comment explaining why.